iree/test/e2e/tosa_ops/pad.mlir

```diff
@@ -0,0 +1,23 @@
+func @pad_1D_test() attributes { iree.module.export } {
+    %0 = "tosa.const"() { value = dense<42> : tensor<2xi32> } : ()  -> (tensor<2xi32>)
+    %1 = "tosa.const"() { value = dense<[[3, 2]]> : tensor<1x2xi32> } : ()  -> (tensor<1x2xi32>)
+    %result = "tosa.pad"(%0, %1)  : (tensor<2xi32>, tensor<1x2xi32>)  -> (tensor<7xi32>)
```

Comment:
Do we test with "tosa.const"? If they are constants, I think they should/will be folded...

Reply:
Good catch. We actually don't have folders in TOSA so it technically works find but better to make input-1 unfoldable.